README.md

```diff
@@ -19,7 +19,7 @@ Wildfire Predictive Services to support decision making in prevention, preparedn
 
 1. Create `.env` file in `web` using `web/.env.example` as a sample.
 2. Create `.env.docker` file in `api/app` using `api/app/.env.example` as a sample.
-3. Run `docker-compose build` and then `docker-compose up`
+3. Run `docker compose build` and then `docker compose up`
```

Comment:
web/readme has `docker compose` without the - and it's newer so i figured it was correct and changed ths one